packages/clay-card/src/CardWithHorizontal.tsx

```diff
@@ -15,16 +15,14 @@ interface IProps {
 	actions?: React.ComponentProps<typeof ClayDropDownWithItems>['items'];
 
 	/**
-	 * Labels for the aria attributes
+	 * Flag to indicate that all interactions on the card will be disabled.
 	 */
-	ariaLabels?: {
-		dropDownTrigger: string;
-	};
+	disabled?: boolean;
 
 	/**
-	 * Flag to indicate that all interactions on the card will be disabled.
+	 * Props to add to the dropdown trigger element
 	 */
-	disabled?: boolean;
+	dropDownTriggerProps?: React.HTMLAttributes<HTMLButtonElement>;
```

Comment:
Rather than have just aria labels available to the trigger element, I thought it would be better to provide a more robust API to allow more customization on that element. This allows them to add other accessibility attrs as needed.

Reply:
I see, you went a step further and added props for those triggers, nice!